### PR TITLE
Click one queue -> Show corrsponding queue detail window

### PR DESCRIPTION
**Pull request by yuexu-98**
- Click Queue -> show corresponding queue detail window
- Add destroy methods for message list and qm selector

Issue:
- Is it possible to add a button on the QM block to trigger the QM detail windows? (since the current QM consists if multiple block objects, it is difficult to locate.)

### CODE REVIEW

**Comment by phillip-tian** [comment]
One small bug I encountered: when creating a second QM inside our app, is the connection window and the first QM visualization overlap, mouse click in the connection window will be captured by the QM visualization in the background. For example, if I click on the input box for apikey, and the current viewpoint has DEV.QUEUE.1 in the background, the mouse click on the apikey input box would bring up the QM info panel.

File: Assets/Scripts/Navigation/MoveCamera.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ public void DoCalculation()
         //Mouse  camera angle done.  
 
         //Keyboard commands
-        float f = 0.0f;
+        //float f = 0.0f;
```
Comment (Diallo) [inline Assets/Scripts/Navigation/MoveCamera.cs]: Just remove this unused variable completely.

File: Assets/Scripts/Navigation/NavigationController.cs [hunk 2]
```diff
@@ -98,4 +101,14 @@ void GenerateCheckBox()
 
     }
 
+    void DestroyQMSelector() {
+        foreach (Transform child in leftPanelContainer) 
+        {
+            if(child.gameObject.name == "QMSelectorRowItem(Clone)")
```
Comment (Diallo) [inline Assets/Scripts/Navigation/NavigationController.cs:107]: Making this string a constant if it needs to be hardcoded.

File: Assets/Scripts/Navigation/NotificationController.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ public class NotificationController : MonoBehaviour
     public Text messageSuccess, messageError, messageInfo, messageWarning;
 
     string currentDate = (DateTime.Now).ToString();
-    public string example = "hello";
+    //public string example = "hello";
```
Comment (Diallo) [inline Assets/Scripts/Navigation/NotificationController.cs]: Remove.

File: Assets/Scripts/Rendering/QueueManager.cs [hunk 4]
```diff
@@ -20,9 +21,11 @@ public class QueueManager : MonoBehaviour
   
     public Dictionary<string, Vector3> offsets;
     public Dictionary<string, int> numberOfRenderedQueues;
-    public Dictionary<string, int[]> dimensions;
-
-    // Unity calls this method at the complete beginning, even before Start
+    public Dictionary<string, int[]> dimensions;
+
+    public static QMDetailsController QMDetailWindow;
```
Comment (Diallo) [inline Assets/Scripts/Rendering/QueueManager.cs:28]: Make this not static.

File: Assets/Scripts/Rendering/QueueManager.cs [hunk 5]
```diff
@@ -90,18 +93,22 @@ of queues.
...
-                GameObject lowerBlock = Instantiate(blockPrefab, new Vector3(sXZ * x, 0, sXZ * z), Quaternion.identity);
+                GameObject lowerBlock = Instantiate(blockPrefab, new Vector3(sXZ * x, 0, sXZ * z), Quaternion.identity)as GameObject;
                 lowerBlock.transform.parent = this.transform;
+                lowerBlock.name = this.name+"Block";
 
                 // Large area
-                GameObject upperBlock = Instantiate(blockPrefab, new Vector3(sXZ * x, 0, sXZ * z) + offsets[numberOfQueuesList[2].Key], Quaternion.identity);
+                GameObject upperBlock = Instantiate(blockPrefab, new Vector3(sXZ * x, 0, sXZ * z) + offsets[numberOfQueuesList[2].Key], Quaternion.identity)as GameObject;
                 upperBlock.transform.parent = this.transform;
+                upperBlock.name = this.name+"Block";
```
Comment (Diallo) [inline Assets/Scripts/Rendering/QueueManager.cs]: General comment: Make this and all strings constants if needed such that it's evident what we use.

File: Assets/Scripts/Rendering/QueueManager.cs [hunk 6]
```diff
@@ -197,21 +207,39 @@ of queues.
...
+                QMDetailWindow.QueueManagerInfoInit(this.name);
+            }
+           
+        }
 
     }
 
+    private void OnMouseDown() 
+    {
+        Debug.Log("TESTING");
```
Comment (Diallo) [inline Assets/Scripts/Rendering/QueueManager.cs]: Remove

File: Assets/Scripts/Rendering/QueueManager.cs [hunk 7]
```diff
@@ -197,21 +207,39 @@ of queues.
...
 
     void Update()
     {
+        if (Input.GetMouseButtonDown(0)) 
+        {
+            Vector3 mousePos = Input.mousePosition;
+
+            if (mousePos.y == 0f)
+            {
+                Debug.Log("HIHI");
```
Comment (Diallo) [inline Assets/Scripts/Rendering/QueueManager.cs]: Remove

**Review by lukasotocerny — DISMISSED** [review]
LGTM

**Comment by yuexu-98** [comment]
> LGTM

Sorry. There is a conflict with the main branch. Need re-review. 😂

**Review by lukasotocerny — APPROVED** [review]
LGTM

**Pull request merged**